deploy/services-demo/conf/galley.demo.yaml

```diff
@@ -39,6 +39,10 @@ settings:
         config:
           enforceAppLock: false
           inactivityTimeoutSecs: 60
+    classifiedDomains:
+      status: enabled
+      config:
+        domains: ["wire.com"]
```

Comment:
```suggestion
        domains: ["example.com"]
```

We should try to avoid putting wire.com in the code.

Reply:
Done.